src/components/dreamOrbsModal.html

```diff
@@ -0,0 +1,28 @@
+<div class="modal fade noselect" id="dreamOrbsModal" tabindex="-1" role="dialog"
+aria-labelledby="dreamOrbsModalLabel">
+<div class="modal-dialog modal-dialog-scrollable" role="document">
+    <div class="modal-content">
+        <div class="modal-header">
+            <h5 class="modal-title">Open Dream Orbs</h5>
+        </div>
+        <div class="modal-body">
+            <select id="dream-orbs-selector" class="custom-select" onchange="App.game.dreamOrbController.selectedOrb(App.game.dreamOrbController.orbs[this.value]); App.game.dreamOrbController.item(undefined)" style="margin-right: 8px">
+                <!-- ko foreach: App.game.dreamOrbController.orbs.filter((o) => !o.requirement ||o.requirement.isCompleted()) -->
+                <option data-bind="attr: { value: $index }, text: $data.color"></option>
+                <!-- /ko -->
+            </select>
```

Comment:
A neater way I've found for writing these select lists.
The filter thing also appears in a couple of places, may be worth making it a computed property on dreamOrbController to make sure everywhere agrees.
```suggestion
            <select id="dream-orbs-selector" class="custom-select" onchange="App.game.dreamOrbController.item(undefined)" style="margin-right: 8px"
                    data-bind="options: App.game.dreamOrbController.orbs.filter((o) => !o.requirement ||o.requirement.isCompleted()),
                               optionsText: 'color',
                               value: App.game.dreamOrbController.selectedOrb"
            >
            </select>
```